gudrun_classes/gud_file.py

```diff
@@ -24,9 +24,9 @@ class GudFile:
         Author of the run.
     stamp : str
         Date and time that the run was completed.
-    densityAcm3 : float
+    atomicDensity : float
         Number density of sample (atoms/Angstrom^3).
-    densityGcm3 : float
+    chemicalDensity : float
         Number density of sample (g/cm^3).
```

Comment:
```suggestion
        Chemical density of sample (g/cm^3).
```
"Number density" === "atomic density"